tests/src/kinAccount.intg.test.ts

```diff
@@ -0,0 +1,156 @@
+		const keyPairs = await Channels.createChannels({
+			environment: Environment.Testnet,
+			baseSeed: thirdKeypair.seed,
+			salt: "salt salt",
+			channelsCount: 2,
+			startingBalance: 1000
+		});
+
+		const account3 = client.createKinAccount({
+			seed: thirdKeypair.seed, channelSecretKeys: keyPairs.map(keypair => {
+				return keypair.seed;
+			})
+		});
+		await account3.channelsPool!!.acquireChannel(async channel => {
+			const builder = await account3.buildSendKin({
+				address: seconedKeypair.publicAddress,
+				amount: 200,
+				fee: 100,
+				memoText: "Send with channels",
+				channel: channel
+			});
+			await account3.submitTransaction(builder);
+			const history2 = await client.getRawTransactionHistory({
+				address: seconedKeypair.publicAddress,
+				order: "desc"
+			});
+			expect(history2[0].source).toBe(channel.keyPair.publicAddress);
+			expect((history2[0]).operations[0].source).toBe(thirdKeypair.publicAddress);
+			done();
+		});
+
+	}, 50000);
+
+	test("Test create account", async () => {
+		const localKeypair = KeyPair.generate();
+		const txBuilder = await account.buildCreateAccount({
+			fee: 100,
+			startingBalance: 1500,
+			memoText: "Test create account",
+			address: localKeypair.publicAddress
+
+		});
+
+		await account.submitTransaction(txBuilder);
+		const data = await client.isAccountExisting(localKeypair.publicAddress);
+		const balance = await client.getAccountBalance(localKeypair.publicAddress);
+		expect(data).toBe(true);
+		expect(balance).toBe(1500);
+
+	}, 30000);
+
+	test("Test send kin", async () => {
+		const txBuilder = await account.buildSendKin({
+			fee: 100,
+			amount: 150,
+			memoText: "Test create account",
+			address: account2.publicAddress
+
+		});
+
+		const balance = await client.getAccountBalance(keyPair.publicAddress);
+		const secondBalance = await client.getAccountBalance(seconedKeypair.publicAddress);
+		await account.submitTransaction(txBuilder);
+		const balance2 = await client.getAccountBalance(keyPair.publicAddress);
+		const secondBalance2 = await client.getAccountBalance(seconedKeypair.publicAddress);
+		expect(balance2).toBe(balance - 150 - 0.001);
+		expect(secondBalance2).toBe(secondBalance + 150);
+
+	}, 30000);
+
+	test("Test whitelist send kin", async () => {
+		const server = new Server(Environment.Testnet.url);
+		const whitelistKeypair = Keypair.fromSecret("SDH76EUIJRM4LARRAOWPBGEAWJMRXFUDCFNBEBMMIO74AWB3MZJYGJ4J");
+		const whitelistAccount = client.createKinAccount({seed: "SDH76EUIJRM4LARRAOWPBGEAWJMRXFUDCFNBEBMMIO74AWB3MZJYGJ4J"});
+		await client.friendbot({address: "GAJCKSF6YXOS52FIIP5MWQY2NGZLCG6RDEKYACETVRA7XV72QRHUKYBJ", amount: 250});
+		const txBuilder = await whitelistAccount.buildSendKin({
+			amount: 250,
+			memoText: "Test create account",
+			address: account2.publicAddress,
+			fee: 0
+		});
+
+		const envelop = txBuilder.build().toEnvelope().toXDR("base64").toString();
```

Comment:
the test should build a transaction from a different account, and then whitelisted it by the whitelisted account, in this case, we're sending the transaction from a whitelisted account so in any case fee will not be deducted.